src/main/java/com/puppycrawl/tools/checkstyle/site/PropertiesMacro.java

```diff
@@ -158,13 +159,32 @@ private static void writeTablePropertiesRows(Sink sink)
         final Map<String, DetailNode> propertiesJavadocs = SiteUtil
                 .getPropertiesJavadocs(properties, currentModuleName, currentModuleFile);
 
-        for (String property : properties) {
+        final Set<String> validProperties = new LinkedHashSet<>(properties);
+
+        orderProperties(validProperties);
```

Comment:
Make it like:
`final Set<String> validProperties = orderProperties(properties);`

Reply:
Done @romani